File: python/easy/jewels_and_stones.py
# 771. Jewels and Stones
# https://leetcode.com/problems/jewels-and-stones/

class Solution:
    def numJewelsInStones(self, jewels: str, stones: str) -> int:
        count = 0
        for i in stones:
            if i in jewels:
                count = count + 1
        return count

File: javascript/easy/jewels_and_stones.js
// 771. Jewels and Stones
// https://leetcode.com/problems/jewels-and-stones/

var numJewelsInStones = function(jewels, stones) {
    var count = 0;
    
    for (var i = 0; i < stones.length; i++) {
        if ( jewels.includes(stones[i]) ) {
            count ++;
        }
    }
    
    return count;
};